walkin/tests/test_management_commands.py

```diff
@@ -0,0 +1,134 @@
+from datetime import timedelta
+
+from django.core.management import call_command
+from django.utils import timezone
+
+from opal.core.test import OpalTestCase
+from elcid.models import PresentingComplaint
+from walkin.models import Symptom
+
+
+class SymptomToPresentingComplaintTest(OpalTestCase):
```

Comment:
Do we test the M2M `symptoms` field gets moved across anywhere?

Reply:
I think we're also not testing the fk_id element.

So we need to do something along the lines of

```
from opal import models as omodels

omodels.Symptom.objects.create(name="some_symptom")

```
The set the fkft field to "some_symptom". ie `self.symptom1.symptom = "some_symptom"; self.symptom1.save()`
(don't actually do this on one line its unpythonic)

Under the covers this will set `symptom_fk` so you can then save that.

Then we need to do something similar but for m2m fields. (ie add it to the `symptoms` field) and check its copied accross.